code/datums/status_effects/buffs.dm

```diff
@@ -62,6 +62,10 @@
 	id = "shadow_mend"
 	duration = 3 SECONDS
 	alert_type = /atom/movable/screen/alert/status_effect/shadow_mend
+	var/devil = FALSE
```

Comment:
Could you autodoc what this does?

Reply:
man I could have sworn I di- oh right that was blood drunk